packages/gestalt/src/ErrorFlyout/__tests__/ErrorFlyout.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
```

Comment:
I had a discussion with @chrislloyd about this and I'm not too convinced that in our testing setup this is the best solution. My understanding is that we currently run the same test file for both a `jsdom` env and a `node` env. By setting this flag here in the doc block up top, we are overriding the env to be `jsdom` for both test runs. Now that works fine to pass CI because it means both runs get the correct env, but if you look in the output, itll still say `node PASS: ErrorFlyout.test.js` which to me means we are duplicating the jsdom test twice and not actually running this as a node test ever despite it being output as one. My hope was that we could put tests that are only relevant to jsdom environments somewhere special and not even have them picked up by the node test so we dont duplicate, while keeping the tests that can be run in both as they are. Check out #205 for my proposal on how to run `jsdom` specific tests. What do you and @chrislloyd think?

Reply:
@pfarejowicz totally agree and noticed that too (the test running twice in the jsdom env). Doesn't seem a blocker for this PR but would be good to fix moving forward.